src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -138,22 +138,8 @@ protected bool LogAapt2EventsFromOutput (string singleLine, MessageImportance me
 					LogCodedError ("APT0001", Properties.Resources.APT0001, message.Substring ("unknown option '".Length).TrimEnd ('.', '\''));
 					return false;
 				}
-				if (message.Contains ("in APK") && message.Contains ("is compressed.")) {
-					LogMessage (singleLine, messageImportance);
+				if (LogNotesOrWarnings (message, singleLine, messageImportance))
```

Comment:
Looking at this reminds me of https://github.com/xamarin/xamarin-android-tools/issues/208 and that `<Aapt2/>` should likewise collect *all* `aapt2` output so that everything can be reported when a warning or error occurs.  It doesn't look like it currently "collects and forwards" all output.

Reply:
I'd like to deal with this in a seperate PR if I can. It might require a bit more rework than I'd be comfortable with putting in this PR.

Reply:
Separate PR is fine.